apis/kueue/v1alpha2/clusterqueue_types.go

```diff
@@ -134,6 +134,7 @@ type ClusterQueueSpec struct {
 	// admitting newer workloads that fit existing quota.
 	//
 	// +kubebuilder:default=BestEffortFIFO
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="QueueingStrategy is immutable"
```

Comment:
I prefer we don't add the CEL validations while we still have the webhooks.

Unless we want to allow users to install kueue without webhooks. But then, we should have **all** the validations implemented in CEL, not just this one.